### ISSUE TITLE
How to apply an IBL texture with any file name?

### ISSUE THREAD

**Issue by cx20** [but0n/Ashes#45]
I tried applying an `IBL` texture to glTF using `Ashes`.
https://cx20.github.io/gltf-test/examples/ashes3d/index.html?category=tutorialModels&model=BoomBox&scale=100&type=glTF

IBL texture can be load with the following file names, but is it possible to use other file names?

```
helipad/<pos|neg><x|y|z>.hdr
```

I want to use the following files for IBL textures.

https://github.com/ux3d/glTF-Sample-Environments/tree/4eace30f795fa77f6e059e3b31aa640c08a82133/papermill/specular

|before          |after                         |
|:---------------|:-----------------------------|
|helipad/posx.hdr|specular/specular_right_0.hdr |
|helipad/negx.hdr|specular/specular_left_0.hdr  |
|helipad/posy.hdr|specular/specular_top_0.hdr   |
|helipad/negy.hdr|specular/specular_bottom_0.hdr|
|helipad/posz.hdr|specular/specular_front_0.hdr |
|helipad/negz.hdr|specular/specular_back_0.hdr  |

**Comment by but0n** [comment]
You can customise file names by modify `Asset.cubemapOrder`

https://github.com/but0n/Ashes/blob/master/src/asset.ts#L284-L291

**Comment by cx20** [comment]
Thank you for telling me how to change the IBL texture.
However, the result of the applied IBL texture seems incorrect.

![image](https://user-images.githubusercontent.com/1833979/77544885-fb8ed300-6eec-11ea-9fbd-0fac7805a006.png)

![image](https://user-images.githubusercontent.com/1833979/77544797-e023c800-6eec-11ea-8795-c7580ae32894.png)

![image](https://user-images.githubusercontent.com/1833979/77544843-efa31100-6eec-11ea-8520-8e538802109c.png)

**Comment by cx20** [comment]
The cause is probably due to mistakenly specifying the same texture as `specEnv` for `diffEnv`.
Although not strictly correct, an improvement was seen by specifying the same texture as `helipad` instead of `papermill` in `diffEnv`.
[Ashes + IBL result:](https://rawcdn.githack.com/cx20/gltf-test/c50eafecebd3c444e175afbec8d3c908ad7bcaa5/examples/ashes3d/index.html?category=tutorialModels&model=BoomBox&scale=100&type=glTF)
![image](https://user-images.githubusercontent.com/1833979/77589001-467d0a80-6f2e-11ea-84a6-fd4a41bd4317.png)

**Comment by cx20** [comment]
I modified the sample because I found a diffuse texture for `papermill`.
https://github.com/ux3d/glTF-Sample-Environments/tree/4eace30f795fa77f6e059e3b31aa640c08a82133/papermill/diffuse

[Ashes + IBL result:](https://rawcdn.githack.com/cx20/gltf-test/1ab4cffaa2d2dd2de7d3c76f264b930ed5e04441/examples/ashes3d/index.html?category=tutorialModels&model=BoomBox&scale=100&type=glTF)
![image](https://user-images.githubusercontent.com/1833979/77593311-5c42fd80-6f37-11ea-958c-76844e9bcf0c.png)

Is this the right way to use it?

**Comment by but0n** [comment]
Thanks for your feedback!
> Is this the right way to use it?

I think this is the right way to use it, but the result is still not strictly correct because of the blur level of specular environment texture, which is a known issue.
It will be improved by using different blur cube textures in next version.

**Comment by cx20** [comment]
Thank you for telling me how to use IBL correctly. I will wait for the next version for specular environment texture issues.

BTW, Please tell me another one. How can I apply a CubeMap Texture to the background using Ashes instead of IBL as follows:

![image](https://user-images.githubusercontent.com/1833979/77643617-0efc7580-6fa3-11ea-8477-feb23c2ad5d2.png)

If that is feasible, I would like to update the gltf-test sample to use CubeMap like other libraries.

**Comment by but0n** [comment]
Do you mean visual the skybox in the scene?
I will handle this feature in next version!

**Comment by cx20** [comment]
I want to test again when the next version comes out. This issue is closed because I understand how to use the IBL texture.

**Comment by but0n** [comment]
This [example](https://but0n.github.io/Ashes/demo.html) shows how to create background using Ashes.
Source: https://github.com/but0n/Ashes/blob/master/demo.html#L104

Also, the HDR specular environment texture issues should be fixed in 0.3.1.

```js
let bg = new Background(screen);
```
![image](https://user-images.githubusercontent.com/7625588/87914700-62e01c80-cab4-11ea-80a1-405b7a257d22.png)


```js
let bg = new Background(screen, skybox);
```
![image](https://user-images.githubusercontent.com/7625588/87914940-cff3b200-cab4-11ea-9d77-4ee1d31a06a5.png)
